python/sglang/srt/custom_op.py

```diff
@@ -20,7 +20,7 @@ def forward_cuda(self, *args, **kwargs):
         raise NotImplementedError
 
     def forward_hip(self, *args, **kwargs):
-        raise NotImplementedError
+        return self.forward_native(*args, **kwargs)
```

Comment:
I can help merge this fix first. Could you submit another PR for that?

Reply:
fine, I will update this PR in a min